relay-config/src/config.rs

```diff
@@ -533,6 +539,8 @@ impl Default for Http {
             connection_timeout: 3,
             max_retry_interval: 60,
             host_header: None,
+            auth_interval: 60,
```

Comment:
As long as we don't have all PRs impld and merged I would make the entire new behavior opt-in.

Reply:
Approved